src/functions-reference/covariance_matrix_distributions.Rmd

```diff
@@ -110,6 +111,52 @@ lower-triangular Cholesky factor of the scale matrix `L_S`; may only be used in
 transformed data and generated quantities blocks
 `r since("2.30")`
 
+## Inverse Wishart distribution, Cholesky Parameterization (#inverse-wishart-cholesky-distribution)
+
+### Probability density function
+
+If $K \in \mathbb{N}$, $\nu \in (K-1, \infty)$, and $L_S \in
+\mathbb{R}^{K \times K}$ is lower triangular and such that $L_S L_S^{\top}$
+is positive definite,then for
+$L_W \in \mathbb{R}^{K \times K}$ is lower triangular and such that $L_WL_W^{\top}$
+is positive definite, \[
+\text{InvWishartCholesky}(L_W \mid \nu,L_S) =
+\text{InvWishart}(W \mid \mu, LL^{\top}). \] If $L$ such that $L \in {L_S, L_W}$  is lower triangular and
+$LL^{top}$ is a $K \times K$ positive definite matrix, then $L_{k, k}$
+must be strictly positive for $k \in 1{:}K$. If an $L$ is provided
+that is not the Cholesky factor of a positive-definite matrix, the
+probability functions will raise errors.
+
+### Stan functions
+
+<!-- real; inv_wishart_cholesky_lpdf; (matrix L_W | real nu, matrix L_S); -->
+\index{{\tt \bfseries inv\_wishart\_cholesky\_lpdf }!{\tt (matrix L\_W \textbar\ real nu, matrix L\_S): real}|hyperpage}
+
+`real` **`inv_wishart_cholesky_lpdf`**`(matrix L_W | real nu, matrix L_S)`<br>\newline
+The log of the inverse Wishart density for lower-triangular Cholesky factor `L_W`
+given degrees of freedom nu and lower-triangular Cholesky factor of the
+scale matrix `L_S`. The lower-triangular Cholesky factors require
+the upper-triangle to be filled with zeros.
+`r since("2.30")`
+
+<!-- real; inv_wishart_cholesky_lupdf; (matrix L_W | real nu, matrix L_S); -->
+\index{{\tt \bfseries inv\_wishart\_lupdf }!{\tt (matrix L\_W \textbar\ real nu, matrix L\_S): real}|hyperpage}
+
+`real` **`inv_wishart_cholesky_lupdf`**`(matrix L_W | real nu, matrix L_S)`<br>\newline
+The log of the inverse Wishart density for lower-triangular Cholesky factor of `L_W`
+given degrees of freedom nu and lower-triangular Cholesky factor of the
+scale matrix `L_S` dropping constant additive terms
+`r since("2.30")`
+
+<!-- matrix; inv_wishart_cholesky_rng; (real nu, matrix L_S); -->
+\index{{\tt \bfseries inv\_wishart\_rng }!{\tt (real nu, matrix L\_S): matrix}|hyperpage}
+
+`matrix` **`inv_wishart_cholesky_rng`**`(real nu, matrix L_S)`<br>\newline
+Generate the Cholesky factor of a inverse Wishart variate with degrees of freedom nu and
+lower-triangular Cholesky factor of the scale matrix `L_S`; may only be used in
+transformed data and generated quantities blocks
```

Comment:
Sentences should end with periods.  To make the rest sentences, it needs to say "return the log of the inverse Wishart density", which is the standard way to write software doc (as a verb).  I understand most of our doc isn't following this pattern, but it should be!